src/Application/FileProcessor.php

```diff
@@ -85,7 +85,10 @@ public function processFile(File $file, Configuration $configuration): FileProce
             // when changed, set final status changed to true
             // to ensure it make sense to verify in next process when needed
             $file->changeHasChanged(true);
+        }
 
+        $rectorWithLineChanges = $file->getRectorWithLineChanges();
```

Comment:
If we have applied any rector rule, we create a FileDiff object even if the code has not actually changed. This FileDiff object will have an empty diff.